projects/1.18/assets/untamedwilds/untamedwilds/lang/zh_cn.json

```
@@ -0,0 +1,726 @@
+  "entity.untamedwilds.tarantula_rose.sciname": "Grammostola rosea",
+  "item.untamedwilds.bottle_tarantula_tiger": "瓶装哥斯达黎加老虎尾蜘蛛",
+  "entity.untamedwilds.tarantula_tiger": "哥斯达黎加老虎尾蜘蛛",
+  "entity.untamedwilds.tarantula_tiger.sciname": "Davus fasciatus",
+  "item.untamedwilds.egg_tarantula": "狼蛛卵",
+  "untamedwilds.subtitles.tarantula.ambient": "狼蛛：爬行",
+
+  "item.untamedwilds.king_crab_spawn_egg": "皇帝蟹刷怪蛋",
+  "entity.untamedwilds.king_crab": "皇帝蟹",
+  "entity.untamedwilds.king_crab_centolla": "南方帝王蟹",
+  "entity.untamedwilds.king_crab_centolla.sciname": "Lithodes santolla",
+  "entity.untamedwilds.king_crab_red": "堪察加拟石蟹",
+  "entity.untamedwilds.king_crab_red.sciname": "Paralithodes camtschaticus",
+  "entity.untamedwilds.king_crab_spiny": "刺鲎",
+  "entity.untamedwilds.king_crab_spiny.sciname": "Paralithodes rathbuni",
+  "item.untamedwilds.egg_king_crab": "皇帝蟹蛋",
+  "untamedwilds.subtitles.king_crab.ambient": "皇帝蟹：爬行",
+  "item.untamedwilds.bucket_king_crab_centolla": "桶装南方帝王蟹",
+  "item.untamedwilds.bucket_king_crab_red": "桶装堪察加拟石蟹",
+  "item.untamedwilds.bucket_king_crab_spiny": "桶装刺鲎",
+
+  "item.untamedwilds.butterfly_spawn_egg": "蝴蝶刷怪蛋",
+  "entity.untamedwilds.butterfly": "蝴蝶",
+  "item.untamedwilds.bottle_butterfly_birdwing": "瓶装亚历山大鸟翼凤蝶",
+  "entity.untamedwilds.butterfly_birdwing": "亚历山大鸟翼凤蝶",
+  "entity.untamedwilds.butterfly_birdwing.sciname": "Ornithoptera alexandrae",
+  "item.untamedwilds.bottle_butterfly_blue_swallowtail": "瓶装美洲蓝凤蝶",
+  "entity.untamedwilds.butterfly_blue_swallowtail": "美洲蓝凤蝶",
+  "entity.untamedwilds.butterfly_blue_swallowtail.sciname": "Battus philenor",
+  "item.untamedwilds.bottle_butterfly_brimstone": "瓶装钩粉蝶",
+  "entity.untamedwilds.butterfly_brimstone": "钩粉蝶",
+  "entity.untamedwilds.butterfly_brimstone.sciname": "Gonepteryx rhamni",
+  "item.untamedwilds.bottle_butterfly_cabbage": "瓶装菜白蝶",
+  "entity.untamedwilds.butterfly_cabbage": "菜白蝶",
+  "entity.untamedwilds.butterfly_cabbage.sciname": "Pieris rapae",
+  "item.untamedwilds.bottle_butterfly_comma": "瓶装白钩蛱蝶",
+  "entity.untamedwilds.butterfly_comma": "白钩蛱蝶",
+  "entity.untamedwilds.butterfly_comma.sciname": "Polygonia c-album",
+  "item.untamedwilds.bottle_butterfly_emperor": "瓶装紫闪蛱蝶",
+  "entity.untamedwilds.butterfly_emperor": "紫闪蛱蝶",
+  "entity.untamedwilds.butterfly_emperor.sciname": "Apatura iris",
+  "item.untamedwilds.bottle_butterfly_glasswing": "瓶装宽纹黑脉绡蝶",
+  "entity.untamedwilds.butterfly_glasswing": "宽纹黑脉绡蝶",
+  "entity.untamedwilds.butterfly_glasswing.sciname": "Greta oto",
+  "item.untamedwilds.bottle_butterfly_kite": "瓶装大帛斑蝶",
+  "entity.untamedwilds.butterfly_kite": "大帛斑蝶",
+  "entity.untamedwilds.butterfly_kite.sciname": "Idea leuconoe",
+  "item.untamedwilds.bottle_butterfly_leaf": "瓶装枯叶蝶",
+  "entity.untamedwilds.butterfly_leaf": "枯叶蝶",
+  "entity.untamedwilds.butterfly_leaf.sciname": "Kallima inachus",
+  "item.untamedwilds.bottle_butterfly_malachite": "瓶装孔雀石蝴蝶",
+  "entity.untamedwilds.butterfly_malachite": "孔雀石蝴蝶",

```

```suggestion
  "item.untamedwilds.bottle_butterfly_malachite": "瓶装绿帘蛱蝶",
  "entity.untamedwilds.butterfly_malachite": "绿帘蛱蝶",
```
https://www.inaturalist.org/taxa/50336-Siproeta-stelenes
不过也有`孔雀石蝶`的称呼